server.go

```diff
@@ -3623,6 +3719,23 @@ func (s *server) prunePersistentPeerConnection(compressedPubKey [33]byte) {
 	s.mu.Unlock()
 }
 
+// bannedPersistentPeerConnection does not actually "ban" a persistent peer. It
```

Comment:
so if it's not banning why do we prefix it with banned ?

Reply:
It's only run for banned peers, I think the naming here is fine and I don't think we need to get lost in the weeds changing the names